src/controllers/groupe.controller.ts

```diff
@@ -0,0 +1,83 @@
+import { NextFunction, Request, Response } from 'express';
+import { Group } from '@prisma/client';
+import { CreateGroupDto } from '@dtos/group.dto';
+import groupService from '@services/groups.service';
+
+class GroupController {
+  public groupService = new groupService();
+
+  public getGroups = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+    try {
+      const findAllGroupsData: Group[] = await this.groupService.findAllGroup();
+
+      res.status(200).json({ data: findAllGroupsData, message: 'findAll' });
+    } catch (error) {
+      next(error);
+    }
+  };
+
+  public getGroupById = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+    try {
+      const groupId = Number(req.params.id);
+      const findOneGroupData: Group = await this.groupService.findGroupById(groupId);
+
+      res.status(200).json({ data: findOneGroupData, message: 'findOne' });
+    } catch (error) {
+      next(error);
+    }
+  };
+
+  public createGroup = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+    try {
+      const groupData: CreateGroupDto = req.body;
```

Comment:
Please add validation eg.

```suggestion
      const groupData: CreateGroupDto = plainToClass(CreateGroupDto, req.body);
      const errors = await validate(groupData);
      if (errors.length > 0) {
        res.status(400).json({ errors: errors.map(error => error.constraints), message: 'Validation error' });
        return;
      }
```